frontend/src/views/applications/OperatingCostsView.vue

```diff
@@ -140,10 +164,54 @@ export default {
       } catch (error) {
         this.setFailureAlert('Failed to save your application', error)
       } finally {
+        this.processing = false
+        this.$emit('process', false)
+      }
+    },
+
+    async getDocuments() {
+      if (this.model.facilityType != FACILITY_TYPES.RENT_LEASE) return
+      try {
+        this.$emit('process', true)
+        this.processing = true
+        this.uploadedDocuments = await DocumentService.getDocuments(this.$route.params.applicationGuid)
+      } catch (error) {
+        this.setFailureAlert('Failed to get your document', error)
```

Comment:
Maybe "Failed to retrieve Supporting Documents"?

Reply:
it is updated